### PR TITLE
[CMSSW_11_3_X] Add new Run3 offline GT in autoCond and its relval workflow

### PR DESCRIPTION
**Pull request by tvami**
#### PR description:

Backport of https://github.com/cms-sw/cmssw/pull/35202

### CODE REVIEW

**Comment by tvami** [comment]
Backport of https://github.com/cms-sw/cmssw/pull/35202

**Comment by tvami** [comment]
test parameters:
  - workflows = 138.1,136.897

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
@cmsbuild , please test

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 1]
```diff
@@ -426,12 +426,15 @@
...
+### run 2021 CRUZET ###
+workflows[136.897] = ['',['RunCosmics2021CRUZET','RECOCOSDRUN3','ALCACOSDRUN3','HARVESTDCRUN3']]
+
 # multi-run harvesting
 workflows[137.8] = ['',['RunEGamma2018C','HLTDR2_2018','RECODR2_2018reHLT_skimEGamma_Offline_L1TEgDQM',
                         'RunEGamma2018D','HLTDR2_2018','RECODR2_2018reHLT_skimEGamma_Prompt_L1TEgDQM','HARVEST2018_L1TEgDQM_MULTIRUN']]
 ### LS2 - MWGR ###
-workflows[138.1] = ['',['RunCosmics2020','RECOCOSDRUN3','ALCACOSDRUN3','HARVESTDCRUN3']]
-workflows[138.2] = ['',['RunCosmics2020','RECOCOSDEXPRUN3','ALCACOSDEXPRUN3','HARVESTDCEXPRUN3']]
+workflows[138.1] = ['',['RunCosmics2021','RECOCOSDPROMPTRUN3','ALCACOSDPROMPTRUN3','HARVESTDCPROMPTRUN3']]
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_standard.py:436]: It seems RunCosmics2021 is not defined in relval_steps.py

Comment (srimanob) [reply Configuration/PyReleaseValidation/python/relval_standard.py:436]: Hi @tvami 
You will need https://github.com/cms-sw/cmssw/pull/34643

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
@cmsbuild, please test

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by srimanob** [comment]
+Upgrade

**Comment by tvami** [comment]
+alca

**Comment by tvami** [comment]
Hi @kskovpen @cms-sw/pdmv-l2 would you mind signing this backport as well? Thanks!

**Comment by jordan-martins** [comment]
+1

**Comment by kskovpen** [comment]
+1

**Comment by perrotta** [comment]
+1
- Meant to technical global runs at T0, while waiting to switch to 12_0_X

**Pull request merged**